ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/MGMT_GATEWAY_TESTS.yml

```diff
@@ -0,0 +1,24 @@
+---
+# tests for mgmt_gateway and ipv6_mgmt_gateway inside node_type (l2leaf)
+
+default_node_types:
+  - node_type: l2leaf
+    match_hostnames:
+      - "TEST-MGMT-GATEWAY-IN-NODE-GROUP"
+
+mgmt_gateway: 192.168.201.250
+ipv6_mgmt_gateway: 2000::1
+
+l2leaf:
+  defaults:
+    mlag_peer_ipv4_pool: 10.10.255.0/24
+    mlag_interfaces: [Ethernet5, Ethernet6]
+  node_groups:
+    - group: MGMT_GATEWAY_TESTS
+      nodes:
+        - name: TEST-MGMT-GATEWAY-IN-NODE-GROUP
+          id: 3
+          mgmt_ip: 192.168.201.202/24
```

Comment:
if we have management ip as IPv4 and management gateway is IPv6 in that case still it takes the Parent IPv4 management gateway
```
- group: LEAFS_DC1
      bgp_as: 65102
      mgmt_gateway: 192.168.66.66
      nodes:
        - name: leaf1-dc1
          id: 3
          mgmt_ip: 192.168.66.105/24
          #ipv6_mgmt_ip: 2001:db8::105/64
          uplink_switch_interfaces: [Ethernet3, Ethernet3]
          ipv6_mgmt_gateway: 2001:db8::4
```

I know this is not ideal case but our cli support it.
```
!
ip route 100.64.0.0/16 100.64.3.1
ip route 172.18.0.0/16 172.18.3.1
ip route vrf MGMT 0.0.0.0/0 192.168.66.1
ip route vrf MGMT 0.0.0.0/0 2001:db8::2
!
ipv6 route vrf MGMT ::/0 192.168.66.1 label 0
!
```
cc: @gmuloc

Reply:
Ufff this does not look good @ClausHolbechArista .

Good catch @MaheshGSLAB

Reply:
ah maybe I am misunderstanding - you are saying that we are not supporting something that can be done in CLI? like having an IPv4 IP in an IPV6 route in `ipv6 route vrf MGMT ::/0 192.168.66.1 label 0` ?

Reply:
i am more concerned on having ipv6 gateway in upv4 route.
ip route vrf MGMT 0.0.0.0/0 2001:db8::2

If my mgmt_ip is IPv4 and giving ipv6_mgmt_gateway: with that case

Reply:
As discussed, we do not want to implement this.